plot_matplotlib.py

```diff
@@ -50,8 +50,8 @@ def plot_params(params_to_plot, title):
     handles = []
     time = continuous_monitor_col('time', cm_data)
     for key in params_to_plot:
-      handle, = pyplot.plot(time, continuous_monitor_col(key, cm_data),
-                         label=key)
+      handle = pyplot.plot(time, continuous_monitor_col(key, cm_data),
+                            label=key)[0]
```

Comment:
Nope! Not crazy. I fixed it, then tested at [0] addition to make sure it behaved the same, then forgot to change it back. doh.